scripts/run-doxygen.sh

```diff
@@ -71,6 +71,10 @@ cp ./refman.pdf ${AMICI_PATH}/AMICI_guide.pdf
 grep -v "warning: Unexpected html tag <img> found within <a href=...> context" ${DOXY_WARNING_FILE} > ${DOXY_WARNING_FILE}_tmp
 mv ${DOXY_WARNING_FILE}_tmp ${DOXY_WARNING_FILE}
 
+# suppress doxygen warnings about multiple param sections
+grep -v "has muliple @param documentation sections" ${DOXY_WARNING_FILE} > ${DOXY_WARNING_FILE}_tmp
```

Comment:
```suggestion
grep -v "iple @param documentation sections" ${DOXY_WARNING_FILE} > ${DOXY_WARNING_FILE}_tmp
```

So we are still fine once the typo gets fixed?